Improve PEGELONLINE docs according to new quality scale (#35940)
home-assistant/home-assistant.github.io: source/_integrations/pegel_online.markdown

This integration uses the data from the German Federal Waterways and Shipping Administration (_Wasserstraßen- und Schifffahrtsverwaltung des Bundes_) [PEGELONLINE](https://www.pegelonline.wsv.de/) to provide different [sensors](#sensors), based on the available data of the selected measurement station.

## Data fetching and limitations

The data are polled every 5 minutes. You can add as many measurement stations as want, there is no authentication needed nor any official usage limitation, but keep the "fair use" in mind.

{% include integrations/config_flow.md %}

### Step 1 - Location selection

Select the area, where you want to search for available water measuring stations

{% configuration_basic %}
Latitude:
    description: "The latitude of the center of the search area (_automatically filled by the location picker_)."

>     required: true

Reviewer: ```suggestion
```

Longitude:
    description: "The longitude of the center of the search area (_automatically filled by the location picker_)."

>     required: true

Reviewer: ```suggestion
```

Radius:
    description: "Search radius (_in km_)"

>     required: true

Author: This is ensured by the implementation, which limits the range

>     required: true

Reviewer: ```suggestion
```

{% endconfiguration_basic %}

### Step 2 - Station selection

{% configuration_basic %}
Station:
    description: "Select the measurement station you want to add."

>     required: true

Reviewer: ```suggestion
```

| Water level | cm |
| Water temperature | °C |
| Water volume flow | m³/s |

## Usage example

### Flood alert level notification

Create an automation to get notified, when your local river reaches a specific flood alert level.

```yaml
mode: single
triggers:
  - trigger: numeric_state
    entity_id:
      - sensor.dresden_elbe_water_level
    above: 500
actions:
  - action: notify.persistent_notification
    metadata: {}
    data:
      message: Flood alert level 2 reached!

> ```

Author: the example should be held simple

## Troubleshooting

> In any case, when reporting an issue, please enable [debug logging](/docs/configuration/troubleshooting/#debug-logs-and-diagnostics), restart the integration, and as soon as the issue re-occurs stop the debug logging again (_download of debug log file will start automatically_). Further _if still possible_, please also download the [diagnostics](/integrations/diagnostics) data. If you have collected the debug log and the diagnostics data, provide them with the issue report.

Reviewer: ```suggestion
In any case, when reporting an issue, please enable [debug logging](/docs/configuration/troubleshooting/#debug-logs-and-diagnostics), restart the integration, and as soon as the issue reoccurs stop the debug logging again (_download of debug log file will start automatically_). Further _if still possible_, please also download the [diagnostics](/integrations/diagnostics) data. If you have collected the debug log and the diagnostics data, provide them with the issue report.
```
